Packs/ApiModules/Scripts/MicrosoftGraphMailApiModule/MicrosoftGraphMailApiModule.py

```diff
@@ -612,7 +615,7 @@ def pages_puller(self, response: dict, page_count: int) -> list:
             list: list of all pages
         """
         responses = [response]
-        for i in range(page_count - 1):
+        for _i in range(page_count - 1):
```

Comment:
the `_` marks an unused argument, and `i` an incremental one. So instead of `i` (supposedly used) or `_` (type unclear`) ,`_i` brings the best of both worlds. I suggest keeping it.